shoop/admin/static_src/base/js/jquery.form-submission-attributes.polyfill.js

```diff
@@ -0,0 +1,51 @@
+//https://github.com/mattberkowitz/Form-Submission-Attributes-Polyfill
+
+(function($,undefined) {
+    "use strict";
+
+    var eles = 'input[type="submit"], input[type="image"], button:not([type]), button[type="submit"]',
+        attrs = ['action', 'method', 'enctype', 'target', 'novalidate'];
+
+    var $f; // form to submit
+    $.fn.formSubmissionAttributes = function () {
+        this.each(function () {
+            var $form = $(this),
+                $inputs;
+
+            if (!$form.is('form')) {
+                console.warn('The formSubmissionAttributes funciton is only intended to be executed on forms');
+                return;
+            }
+
+            $inputs = $form.find(eles);
+
+            if ($form.attr('id') && $form.attr('id') !== '') {
```

Comment:
The empty-string comparison isn't necessary.

![image](https://cloud.githubusercontent.com/assets/58669/9301233/a639e462-44d5-11e5-99a7-85bf89fd6927.png)